[2025-08-08T12:23:41.145+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_data_pipeline.validate_environment manual__2025-08-08T12:23:40.425042+00:00 [queued]>
[2025-08-08T12:23:41.178+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_data_pipeline.validate_environment manual__2025-08-08T12:23:40.425042+00:00 [queued]>
[2025-08-08T12:23:41.179+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 4
[2025-08-08T12:23:41.217+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): validate_environment> on 2025-08-08 12:23:40.425042+00:00
[2025-08-08T12:23:41.228+0000] {standard_task_runner.py:60} INFO - Started process 312 to run task
[2025-08-08T12:23:41.237+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'stock_data_pipeline', 'validate_environment', 'manual__2025-08-08T12:23:40.425042+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/stock_data_dag.py', '--cfg-path', '/tmp/tmpmpu7tspm']
[2025-08-08T12:23:41.242+0000] {standard_task_runner.py:88} INFO - Job 8: Subtask validate_environment
[2025-08-08T12:23:41.311+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-08T12:23:41.432+0000] {task_command.py:423} INFO - Running <TaskInstance: stock_data_pipeline.validate_environment manual__2025-08-08T12:23:40.425042+00:00 [running]> on host e0c05113885d
[2025-08-08T12:23:41.610+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_team' AIRFLOW_CTX_DAG_ID='stock_data_pipeline' AIRFLOW_CTX_TASK_ID='validate_environment' AIRFLOW_CTX_EXECUTION_DATE='2025-08-08T12:23:40.425042+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-08T12:23:40.425042+00:00'
[2025-08-08T12:23:41.612+0000] {fetch_and_store.py:34} INFO - All required environment variables are set
[2025-08-08T12:23:41.613+0000] {stock_data_dag.py:37} INFO - Environment validation successful
[2025-08-08T12:23:41.613+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-08-08T12:23:41.633+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=stock_data_pipeline, task_id=validate_environment, execution_date=20250808T122340, start_date=20250808T122341, end_date=20250808T122341
[2025-08-08T12:23:41.697+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-08T12:23:41.731+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
